commands/command_test.go

```diff
@@ -469,3 +472,493 @@ func (s *MmctlUnitTestSuite) TestCommandListCmdF() {
 		s.Equal("Unable to list commands for '"+teamID+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlUnitTestSuite) helperCreateCommand(*model.Command) {
+	s.T().Helper()
+
```

Comment:
Unnecessary whitespace

Reply:
Thanks @mgdelacroix   This was left over from some experimental unit test helper that I missed when cleaning up.  Removed.